File: python_programming_basics/conditional_statements/e_08_lunch_break.py
import math

serial_name = input()
episode_duration = int(input())
rest_duration = int(input())

lunch_time = rest_duration / 8
relax_time = rest_duration / 4

total_time = rest_duration - (lunch_time + relax_time)

if total_time >= episode_duration:
    print(
        f"You have enough time to watch {serial_name} and left with {math.ceil(total_time - episode_duration)} minutes free time.")

else:
    print(
        f"You don't have enough time to watch {serial_name}, you need {math.ceil(episode_duration - total_time)} more minutes.")

File: JavaScript Basics/02. Conditional Statements/e-08-Lunch-Break.js
function solve(name, episodeDuration, restDuration) {
    lunchTime = restDuration / 8;
    relaxTime = restDuration  / 4;

    timeLeft = restDuration - lunchTime - relaxTime;
    
    if (timeLeft >= episodeDuration) {
        time = timeLeft - episodeDuration
        console.log(`You have enough time to watch ${name} and left with ${Math.ceil(time)} minutes free time.`);
        
    } else {
        neededTime = episodeDuration - timeLeft
        console.log(`You don't have enough time to watch ${name}, you need ${Math.ceil(neededTime)} more minutes.`);
        
    } 

}

solve("Game of Thrones", 60, 96)